library_generation/generate_repo.py

```diff
@@ -39,6 +41,16 @@ def generate_from_yaml(
     If specified with an empty list, then no library will be generated.
     If not specified, all libraries in the configuration yaml will be generated.
     """
+    # Determine whether the generated repo is a proto-only repository.
+    # Using libraries in generation config without filtering out target library
+    # names because the remaining library may not qualify as proto-only
+    # library, e.g., iam.
+    proto_only_repo = False
```

Comment:
Can we move this logic to `is_monorepo()`? Or we need it for something else other than the monorepo post-processing?

Reply:
It is used to determine the monorepo name, google-cloud-java if `proto_only_repo` is false; sdk-platform-java otherwise. If we have a third monorepo, we need to revise this logic.

Also, do not populate `api_id` in `.repo-metadata.json` if `proto_only_repo` is `true`. From the `README.md` of common-protos and iam, they don't have an api_id.